app/js/components/orderEntry/ActiveOrders.jsx

```diff
@@ -49,26 +49,26 @@ export class ActiveOrders extends React.Component {
 
       if (dosingType === 'org.openmrs.SimpleDosingInstructions') {
         details = (
-          <td className="fs-14-px">
+          <td>
 
             {drug.display}:
-            {dose && ` ${dose}`}
-            {doseUnits.display && ` ${doseUnits.display}`}
-            {frequency.display && `, ${frequency.display}`}
-            {route.display && `, ${route.display}`}
+            {` ${dose}`}
+            {` ${doseUnits.display}`}
+            {`, ${frequency.display}`}
+            {`, ${route.display}`}
             {duration && `, for ${duration}`}
-            {durationUnits.display && ` ${durationUnits.display} total`}
+            {durationUnits && ` ${durationUnits.display} total`}
             {dosingInstructions && `, (${dosingInstructions})`}
-            {(quantity && quantityUnits.display) && `, (Dispense: ${quantity} ${quantityUnits.display})`}
+            {(quantity && quantityUnits) && `, (Dispense: ${quantity} ${quantityUnits.display})`}
 
           </td>
         );
       } else {
         details = (
-          <td className="fs-14-px">
+          <td>
 
             {drug.display}:
-            {dosingInstructions && ` (${dosingInstructions})`}
+            {dosingInstructions && ` ${dosingInstructions}`}
             {(quantity && quantityUnits.display) && `, (Dispense: ${quantity} ${quantityUnits.display})`}
```

Comment:
For inpatient caresettings; there will be no quantity and quantity units so `quantityUnits.display` will make the app crash